charmhelpers/contrib/openstack/ip.py

```diff
@@ -145,6 +146,26 @@ def local_address(unit_get_fallback='public-address'):
         return unit_get(unit_get_fallback)
 
 
+def get_invalid_vips():
+    """Returns a list of invalid vips.
+    If all vips are valid, returns an empty list.
```

Comment:
Please can you add details of what the function returns and its type (see the local_address function above for an example)